tools/erc-repository-indexer/erc-contract-indexer/erc-registry/mainnet/erc-20.json

```diff
@@ -0,0 +1,3122 @@
+[
+  {
+    "contractId": "0.0.14902",
+    "address": "0x0000000000000000000000000000000000003a36",
+    "name": "",
```

Comment:
Q: Are the empty name and symbols mistakes?

Reply:
It appears that the contract was deployed with these fields left uninitialized or empty or whatnot. 

The contract at address 0x0000000000000000000000000000000000003a36 contains bytecode that passes the sevm.isErc() method and successfully responds to all contract mirror node call requests (name, symbol, decimals, totalSupply) but the return values are all empty and 0s.

Reply:
Gotcha, I guess we'll leave it for users to verify and use as needed.